app/views/shared/authentication/_email_registration_form.html.erb

```diff
@@ -54,6 +59,14 @@
         <%= f.password_field :forem_owner_secret, autocomplete: "current-password", class: "crayons-textfield", required: true, placeholder: "As provided by your Forem host" %>
       </div>
     <% end %>
+
+    <% if SiteConfig.recaptcha_secret_key.present? &&
```

Comment:
**Suggestion**: this is a pretty beefy conditional in a view. Consider extracting this to a helper method.

Reply:
I was very pleased to see this when I showed up to review it. Very clean. 🤙